Makefile

```diff
@@ -69,19 +69,19 @@ gosec:
 
 gen: $(GENSRC)
 
-timestamp-cli: $(SRCS)
-	CGO_ENABLED=0 go build -trimpath -ldflags "$(CLI_LDFLAGS)" -o timestamp-cli ./cmd/timestamp-cli
+bin/timestamp-cli: $(SRCS)
```

Comment:
Can we leave these with the bin prefix? I think so, this is just how to refer to it in other targets if i recall makefiles